lib/stan_math_2.6.3/stan/math/prim/scal/prob/wiener_log.hpp

```diff
@@ -0,0 +1,229 @@
+        x = x / alpha2;  // convert t to normalized time tt
+        T_return_type sqrt_x = sqrt(x);
+
+        // calculate number of terms needed for large t:
+        // if error threshold is set low enough
+        if (pi() * x * WIENER_ERR < 1) {
+          // compute bound
+          kl = sqrt(-2.0 * log(pi() * x * WIENER_ERR) * SQRT_PI) /
+               sqrt_x;
+          // ensure boundary conditions met
+          kl = (kl > 1.0 / (pi() * sqrt_x)) ?
+               kl : 1.0 / (pi() * sqrt_x);
+        } else {  // if error threshold set too high
+          kl = 1.0 / (pi() * sqrt_x);  // set to boundary condition
+        }
+        // calculate number of terms needed for small t:
+        // if error threshold is set low enough
+        T_return_type tmp_expr0
+          = 2.0 * SQRT_2_TIMES_SQRT_PI * sqrt_x * WIENER_ERR;
+        if (tmp_expr0 < 1) {
+          // compute bound
+          ks = 2.0 +  sqrt_x * sqrt(-2 * log(tmp_expr0));
+          // ensure boundary conditions are met
+          T_return_type sqrt_x_plus_one = sqrt_x + 1.0;
+          ks = (ks > sqrt_x_plus_one) ? ks : sqrt_x_plus_one;
+        } else {  // if error threshold was set too high
+          ks = 2.0;  // minimal kappa for that case
+        }
+        // compute density: f(tt|0,1,w)
+        if (ks < kl) {  // if small t is better (i.e., lambda<0)
+          K = ceil(ks);  // round to smallest integer meeting error
+          T_return_type tmp_expr1 = (K - 1.0) / 2.0;
+          T_return_type tmp_expr2 = ceil(tmp_expr1);
+          for (k = -floor(tmp_expr1); k <= tmp_expr2; k++)
+            // increment sum
+            tmp += (one_minus_beta + 2.0 * k) *
+                    exp(-(square(one_minus_beta + 2.0 * k)) * 0.5 / x);
+            // add constant term
+            tmp = log(tmp) -
+                  LOG_TWO_OVER_TWO_PLUS_LOG_SQRT_PI- 1.5 * log(x);
+        } else {  // if large t is better...
+          K = ceil(kl);  // round to smallest integer meeting error
+          for (k = 1; k <= K; k++)
+            // increment sum
+            tmp += k * exp(-(pow(k, 2)) *
+                   (square(pi()) * 0.5 * x)) *
+                   sin(k * pi() * one_minus_beta);
+            tmp = log(tmp) + 2.0 * LOG_SQRT_PI;  // add constant term
+        }
+
+        // convert to f(t|v,a,w) and return result
+        lp += delta_vec[i] * alpha_vec[i] * one_minus_beta -
+              pow(delta_vec[i], 2) * x * alpha2 / 2.0 -
+              log(alpha2) + tmp;
+      }
+
+      return lp;
+    }
+
+    template <typename T_y, typename T_alpha, typename T_tau,
+              typename T_beta, typename T_delta>
+    inline
+    typename return_type<T_y, T_alpha, T_tau, T_beta, T_delta>::type
+    wiener_log(const T_y& y, const T_alpha& alpha, const T_tau& tau,
+               const T_beta& beta, const T_delta& delta) {
+      return wiener_log<false>(y, alpha, tau, beta, delta);
+    }
+
+    /* Not actually implemented yet
```

Comment:
Don't include commented out code.   Either include it or don't.  You can start a new branch if you don't want to lose it.